src/ankh/cmd/ankh/main.go

```diff
@@ -261,3 +272,52 @@ func check(err error) {
 		os.Exit(1)
 	}
 }
+
+func runScripts(ctx *ankh.ExecutionContext, scripts []struct { Path string }) {
+	for _, script := range scripts {
+		path := script.Path
+		if path == "" {
+			log.Warnf("Missing path in script %s", script)
+			break
+		}
+		info, err := os.Stat(path)
+		if os.IsNotExist(err) {
+			log.Fatalf("Script not found: %s", path)
+			os.Exit(1)
```

Comment:
`log.Fatalf` automatically calls `os.Exit(1)` so this line isn't needed. Looks like that's in a few place.

Reply:
Oh makes sense, I'll fix that